src/tests/test02.vtc

```diff
@@ -2,7 +2,7 @@ varnishtest "Test digest vmod out of workspace"
 
 server s1 { } -start
 
-varnish v1 -arg "-p workspace_client=9k" -vcl+backend {
```

Comment:
So, how much workspace do you need for this to pass? It is interesting to understand how much workspace is now needed for this to work.

In short, maybe a change of `9k `to `37k` (or whatever is needed) is better, so that we will notice when we suddenly need even more workspace to run.

Reply:
You are correct, this should have a separate test case in varnish cache. I think this is ready to merge.

Reply:
You could also first set the `vsl_buffer` parameter to a lower value to keep the current workspace setting.

Reply:
Never mind, the test uses `vtc.workspace_alloc(client, -5);` so it shouldn't need a workspace tweak in the first place.